app/register/template.hbs

```diff
@@ -0,0 +1,26 @@
+{{title (t 'routes.register.sign_up')}}
+<div class="container">
+    <div class="row m-t-xl m-b-xl">
+        <div local-class="sign-up-container" class="p-md bg-color-light">
+            <h3 class="m-sm">{{t 'routes.register.create_a_free_account'}}</h3>
+            <h4 class="m-md">{{t 'routes.register.sign_up_using'}}</h4>
+            <div local-class="sign-up-buttons" class="m-md">
+                <a local-class="sign-up-button" href="{{orcidUrl}}" onclick={{action 'click' 'button' 'Sign up - ORCID' target=analytics}} data-test-orcid-button>
```

Comment:
Minor: I think it's easier to read a bunch of shorter lines than one long line:
```hbs
<a data-test-orcid-button
    href="{{orcidUrl}}"
    local-class="sign-up-button"
    onclick={{action 'click' 'button' 'Sign up - ORCID' target=analytics}}
>
    ...
</a>

Reply:
:+1: